comet/archiver.py

```diff
@@ -6,22 +6,20 @@
 
 import asyncio
 import datetime
+import logging
 import orjson as json
 from pathlib import Path
 import signal
 
-import logging
-
 import chimedb.core as chimedb
 import chimedb.dataset as db
 
 from . import Manager, CometError, __version__
 from .broker import DEFAULT_PORT
-from .manager import TIMESTAMP_FORMAT, LOG_FORMAT
+from .manager import TIMESTAMP_FORMAT
 
 
-logging.basicConfig(format=LOG_FORMAT)
```

Comment:
Why did these get moved to scripts?

Reply:
archiver.py obviously only gets used by the script anyways, but if a project imports `comet.manager` for example and it already has configured logging, it ends up with duplicate logging lines.

Reply:
Gotcha!